src/main/java/com/hillel/items_exchange/service/ProductService.java

```diff
@@ -0,0 +1,18 @@
+package com.hillel.items_exchange.service;
+
+import com.hillel.items_exchange.dao.ProductRepository;
+import com.hillel.items_exchange.model.Product;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+
+@RequiredArgsConstructor
+@Component
```

Comment:
I understand, that @Service = @Component, but would you like to change this for better comprehensibility? =)

Reply:
Yup, I agree and I'll change it.